Worktez/worktez
# fix(create new sprint): "fixed create new sprint message and show details"

### Functionality:
fixed create new sprint 

### Solution:
fixed the create new  sprint message and display the details immediately after we create a new sprint.

### Risk level:

### How to test:
In board page click on create new sprint and check if it show message on creating new sprint
